onpremise/onpremise.go

```diff
@@ -171,25 +178,102 @@ func WithCustomLogger(logger LogWriter) EngineOptions {
 
 		return nil
 	}
+}
+
+// ToggleFileWatch enables or disables file watching
+// in case 3rd party updates the data file on file system
+// engine will automatically reload the data file
+// default is true
+func ToggleFileWatch(enabled bool) EngineOptions {
+	return func(cfg *Engine) error {
+		cfg.isFileWatcherEnabled = enabled
+		return nil
+	}
+}
+
+// ToggleUpdateOnStart enables or disables update on start
+// default is true
```

Comment:
Let's change it to False by default, please, and update  the tests.  For reference implementation has it as False by default: https://github.com/51Degrees/pipeline-dotnet/blob/c75b438d0635a63e0fb562e7736659f61741ba73/FiftyOne.Pipeline.Engines/Constants.cs#L117